### DISCUSSION TITLE
Oh-my-zsh

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1487] [General]
Oh-my-zsh installation uses  git clone for installation for installation which defaults to `~/.oh-my-zsh` directory.
But the aur version uses `/usr/share/oh-my-zsh`. The later one becomes a problem when adding custom zsh plugins for example
https://github.com/mroth/evalcache which uses git clone for installation. This is because write operations on /usr/ folders require elevated permissions

**Comment by rubiin** [comment]
@kRHYME7

**Reply by kRHYME7** [reply]
Try to make a patch for this  and will discuss this. Ig we need tittu to see this.

**Reply by rubiin** [reply]
Some popular hyprland based dotfiles are doing the same.
https://github.com/mylinuxforwork/dotfiles/blob/c78b1a18991c1a61f5b85e61bbc2185bb05f3bdb/share/dotfiles/.config/ml4w/scripts/shell.sh#L57
https://github.com/JaKooLit/Arch-Hyprland/blob/e25cbba2c6362bf18c5dfd9b8d52e61d23c1d4f8/install-scripts/zsh.sh#L64
